src/Lecture1.hs

```diff
@@ -119,4 +136,15 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = go 0 0 list
+  where
+    go :: Int -> Int -> [Int] -> String
+    go nless nge l
+      | null l = show n ++ " is greater than " ++ show nge ++ " elements and lower than " ++ show nless ++ " elements"
+      | n > x = go nless (1 + nge) xs
+      | n < x = go (1 + nless) nge xs
+      | n == x = go nless nge xs
```

Comment:
GHC outputs some warnings due to non-exhaustive pattern-matching ⚠️ 

This is happening because the compiler can't check if all the guards really cover all cases. Usually, to make the compiler happy, the last guard is `otherwise` 💂🏻